src/platforms/ios/ioscontroller.mm

```diff
@@ -160,6 +160,11 @@
       }
       onboardingCompletedCallback:^() {
         MozillaVPN::instance()->onboardingCompleted();
+      }
+      allowVpnConfigOptionSelectedCallback:^() {
+        ConnectionManager* connectionManager = MozillaVPN::instance()->connectionManager();
+        Q_ASSERT(connectionManager);
```

Comment:
Nit: this assert doesn't add much value IMO and the code base doesn't use asserts much when getting singletons.